modules/appeals_api/app/models/concerns/appeals_api/nod_status.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'appeals_api/central_mail_updater'
+
+module AppealsApi
+  module NodStatus
```

Comment:
if we wanted `NODStatus`, we'd have to name the file something like `n_o_d_status`, right? Is there a way to inflect that?

Reply:
I guess it's probably not worth flouting "convention over configuration"

Reply:
Hmm, I'll look into this...

Reply:
If we would make an entry for `NOD` here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/inflections.rb we could use `NODStatus` and still have the file name `nod_status`... Do you think it would be worth it?